lib/screens/v2/widget_instance/elevator_closures.ex

```diff
@@ -119,7 +119,7 @@ defmodule Screens.V2.WidgetInstance.ElevatorClosures do
   @type t :: %__MODULE__{
           app_params: Elevator.t(),
           now: DateTime.t(),
-          stations_with_closures: list(Station.t()),
+          stations_with_closures: list(Station.t()) | :no_closures,
```

Comment:
We could be more specific here now and say `nonempty_list(Station.t())`, since if the list were empty, we would instead use `:no_closures`.